learning/tour-of-beam/frontend/pubspec.lock

```diff
@@ -278,7 +278,7 @@ packages:
       name: flutter_code_editor
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "0.1.1"
+    version: "0.1.3"
```

Comment:
It's already `0.1.3` in `master`. This will likely cause a merge conflict.

Reply:
I rebased the branch on top of `master`.